inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/sort/node/provider/KafkaProvider.java

```diff
@@ -113,11 +98,61 @@ public LoadNode createLoadNode(StreamNode nodeInfo, Map<String, StreamField> con
         Map<String, String> properties = parseProperties(kafkaSink.getProperties());
         List<FieldInfo> fieldInfos = parseSinkFieldInfos(kafkaSink.getSinkFieldList(), kafkaSink.getSinkName());
         List<FieldRelation> fieldRelations = parseSinkFields(kafkaSink.getSinkFieldList(), constantFieldMap);
-        Integer sinkParallelism = null;
-        if (StringUtils.isNotEmpty(kafkaSink.getPartitionNum())) {
-            sinkParallelism = Integer.parseInt(kafkaSink.getPartitionNum());
+
+        String partitionNum = kafkaSink.getPartitionNum();
```

Comment:
`partitionNum` maybe `Integer`？

Reply:
> Integer


partitionNum is defined as String in KafkaSink, theoretically it should be defined as Integer, next modification